utils/reviewsUtils.js

```diff
@@ -75,7 +75,7 @@ export async function getAverageRating(adapter, id, imdbAdapter) {
 }
 
 export async function validateReview(review) {
-  if (!review.data.author || !review.data.rating) {
+  if (!review.data.author || review.data.rating < 0 || review.data.rating > 5) {
```

Comment:
You're adding/changing logic in the function, but there are no changes to the tests to check for this new functionality. I think it would have been good to have one test for checking if an author is missing, one for rating < 0 and one for rating > 5.